Resources/Storage/SRM2Storage.py

```diff
@@ -111,6 +111,16 @@ def __init__( self, storageName, parameters ):
     self.MAX_SINGLE_STREAM_SIZE = 1024 * 1024 * 10  # 10 MB ???
     self.MIN_BANDWIDTH = 0.5 * ( 1024 * 1024 )  # 0.5 MB/s ???
 
+    # Get SE status if needed
```

Comment:
This is not really a nice dependency. I would rather recommend to use the reference to the StorageElement class put in place by Andrei: StorageBase has a self.se attribute